src/app/pages/cars/cars/cars-list-table-data-source.ts

```diff
@@ -226,19 +235,24 @@ export class CarsListTableDataSource extends TableDataSource<Car> {
       case CarButtonAction.CREATE_CAR:
         if (actionDef.action) {
           (actionDef as TableCreateCarActionDef).action(CarDialogComponent,
-            this.dialog, this.refreshData.bind(this));
+            this.dialog, { authorizations: this.authorizationActions }, this.refreshData.bind(this));
```

Comment:
I would have expected only `this.authorizationActions` instead of `{ authorizations: this.authorizationActions }`
It gives the impression that you can pass the Car in addition to this param.